app/models/devise_token_auth/concerns/user.rb

```diff
@@ -42,87 +42,68 @@ def self.tokens_match?(token_hash, token)
     before_save :remove_tokens_after_password_reset
 
     # don't use default devise email validation
-    def email_required?
-      false
-    end
-
-    def email_changed?
-      false
-    end
-
-    def will_save_change_to_email?
-      false
-    end
+    def email_required?; false; end
+    def email_changed?; false; end
+    def will_save_change_to_email?; false; end
```

Comment:
These unexplained `devise` overrides are the bane of my existence and the cause of a lot of hair pulling for a lot of developers...  But alas, you aren't changing their functionality here.  

That said, I'd just leave these alone, so we can preserve our git blame history on these three methods to help the next poor soul who encounters this gotcha while wondering what's magically overriding their email callback on their user object.

Reply:
@KelseyDH is there a pr to fix the devise overrides situation? If not, feel free to open  so we can debate!